conda/recipes/dask-cuda/meta.yaml

```diff
@@ -31,7 +31,7 @@ requirements:
     - distributed >=2.22.0
```

Comment:
I think we may need to constrain `dask` + `distributed` here...

Reply:
But do we need to constrain it here? Shouldn't this be happening RAPIDS-wide?

Reply:
Specifically, I was under the impression that https://github.com/rapidsai/integration/blob/37f52deb0acc7826759cef61f2e5109192fee3db/conda/recipes/versions.yaml#L49-L50 would resolve this. Am I wrong?

Reply:
Yes I think so if we are still seeing that issue.

We do that also

https://github.com/rapidsai/integration/blob/37f52deb0acc7826759cef61f2e5109192fee3db/conda/recipes/versions.yaml#L49-L50
https://github.com/rapidsai/integration/blob/37f52deb0acc7826759cef61f2e5109192fee3db/conda/recipes/versions.yaml#L53-L54

Though I don't think that is working for us here. So we may need to add it ourselves

Reply:
Ok, I'll add here as well, it must probably be by design, as we also release Dask-CUDA PyPI packages we probably don't necessarily want to limit it to the same narrow versions as RAPIDS, but not sure how much that really helps as we don't test against older Dask versions anyway.

Reply:
Yeah makes sense

Good point. Maybe in the next release we can bump the minimum